Cli replace with kong package and removed all global variables (#320)
tellor-io/telliot: docs/changelog.md

## Unreleased

* [\#240](https://github.com/tellor-io/telliot/issues/240) Replaced the cli package to allow for command specific flags and configuration. Now all flags should be passed last. Example:
Instead of: `./telliot --config="config.json" stake deposit`, it becomes:`./telliot stake deposit --config="config.json"`

> 
> * Removed the `RemoteMining` from the config. Remote is active when specifying a `RemoteDBHost` for the `Mine` command. See the `configs/config.json` for an example.

Reviewer: ```suggestion
Removed the `RemoteMining` `-r` flag. Remote is active when specifying a `RemoteDBHost` for the `Mine` command. See the `configs/config.json` for an example.
```

### Changed

### Added

### Fixed

> ## [v5.5.0](https://github.com/tellor-io/telliot/releases/tag/v5.5.0) * 2021.01.18

Reviewer: ```suggestion
## [v5.5.0](https://github.com/tellor-io/telliot/releases/tag/v5.5.0) - 2021.01.18
```

### Changed

> * [\#372](https://github.com/tellor*io/telliot/pull/372) Split the configs of the mine and dataserver command to avoid confusion and be more explicit. This also fixes an issue where can't run a miner and dataserver on the same machine(now that the miner also runs an HTTP server to expose metrics). The config format has changed so users need to update their configs. See the `configs/config.json` for an example of the new format.

Reviewer: ```suggestion
* [\#372](https://github.com/tellor-io/telliot/pull/372) Split the configs of the mine and dataserver command to avoid confusion and be more explicit. This also fixes an issue where can't run a miner and dataserver on the same machine(now that the miner also runs an HTTP server to expose metrics). The config format has changed so users need to update their configs. See the `configs/config.json` for an example of the new format.
```

## Unreleased

- [\#240](https://github.com/tellor-io/telliot/issues/240) Replaced the cli package to allow for command specific flags and configuration. Now all flags should be passed last. Example:
  Instead of: `./telliot --config="config.json" stake deposit`, it becomes:`./telliot stake deposit --config="config.json"`

> 
> - Removed the `RemoteMining` from the config. Remote is active when specifying a `RemoteDBHost` for the `Mine` command. See the `configs/config.json` for an example.
> 

Reviewer: gitbook doesn't like extra white spaces I think.
```suggestion
- Removed the `RemoteMining` from the config. Remote is active when specifying a `RemoteDBHost` for the `Mine` command. See the `configs/config.json` for an example.

```
